run-servers.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $1 = "--ci" ]]; then
+  echo "Launch script finished"
+else
+  trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+fi
+
+
+SERVER_VERSION="10.1.2.Final"
+SERVER_HOME=server/infinispan-server-$SERVER_VERSION
+CLUSTER_SIZE_MAIN="$SERVER_HOME/bin/cli.sh -c localhost:11322 -f batch "
+ZIP_ROOT="http://downloads.jboss.org/infinispan"
+
+CONF_DIR_TO_COPY_FROM="spec/configs/"
+IS_SSL_PROCESSED=0
+SERVER_DIR="infinispan-js-serverng"
+
+function waitForClusters()
+{
+cat > batch<<EOF
+describe
+disconnect
+EOF
+
+  MEMBERS_MAIN=''
+  while [ "$MEMBERS_MAIN" != '3' ];
+  do
+    MEMBERS_MAIN=$($CLUSTER_SIZE_MAIN | grep cluster_size | cut -d':' -f2 | sed 's/.$//' | sed -e 's/^[[:space:]]*//')
+    echo "Waiting for clusters to form (main: $MEMBERS_MAIN)"
+    sleep 20
+  done
+}
+
+function prepareServerDir()
+{
+    local isCi=$1
+    local confPath=$2
+    local isSsl=$3
+    local dirName=${4}
+
+    if [ ! -f server/infinispan-server-$SERVER_VERSION.zip ]; then
+        cd server
+        wget $ZIP_ROOT/$SERVER_VERSION/infinispan-server-$SERVER_VERSION.zip
+        unzip  infinispan-server-$SERVER_VERSION.zip
+        cd ..
+    fi
+
+    if [[ ${isCi} = "--ci" ]]; then
+        SERVER_TMP=${SERVER_HOME}
+        echo "Use server in directory: $SERVER_TMP"
+    else
+        if [[ -z "${SERVER_TMP}" ]]; then
+             SERVER_TMP=${TMPDIR:-'/tmp'}/${SERVER_DIR}
+             mkdir ${SERVER_TMP} 2>/dev/null
+             echo "Created temporary directory: $SERVER_TMP"
+
+             cp -r ${SERVER_HOME}/* $SERVER_TMP
+             echo "Server copied to temporary directory."
+
+             $SERVER_TMP/bin/user-tool.sh -u admin -p 'mypassword'
+             echo "Admin user added."
+        fi
+
+        cp -r ${SERVER_HOME}/server ${SERVER_TMP}/${dirName}
```

Comment:
this copy needs to be done with or without the `--ci` option. 
Otherwise, if you run `run-servers.sh --ci` is will fail to execute.